scripts/lib/setup_venv.py

```diff
@@ -40,7 +40,8 @@
 THUMBOR_VENV_DEPENDENCIES = [
     "libcurl4-openssl-dev",
     "libjpeg-dev",
-    "libjasper-dev",
+    "zlib1g-dev",
```

Comment:
Why this rather than `libz-dev`?

Reply:
when we install `libz-dev` I got this
>Note, selecting 'zlib1g-dev' instead of 'libz-dev'
from apt-get and at this link https://pillow.readthedocs.io/en/latest/installation.html#external-libraries they seem to say
```
Prerequisites are installed on Ubuntu 14.04 LTS with:

$ sudo apt-get install libtiff5-dev libjpeg8-dev zlib1g-dev \
    libfreetype6-dev liblcms2-dev libwebp-dev libharfbuzz-dev libfribidi-dev \
    tcl8.6-dev tk8.6-dev python-tk
```

Also when we use apt-get to install `libz-dev` only `zlib1g-dev` is installed but doing a apt-get install 'zlib1g-dev' installs other deps as well. 
Since Pillow docs mentioned it and I observed difference between deps installed I got inclined towards doing `zlib1g-dev`

Reply:
OK, I did a bit of `apt show` work to investigate, and these two packages are actually the same (`zlib1g-dev` is the new name for `libz-dev` in Debian/Ubuntu; the latter is now just a "virtual" package).  So we probably want to change this in the main Zulip list too, since `zlib1g-dev` is the name that will stick around into the future, I imagine.

Add a commit for this?

Reply:
I can add a new commit for doing the changes Zulip wide, hold on.